backend/tests/endpoints/test_room.py

```diff
@@ -1,4 +1,123 @@
+from unittest import mock
+
+import pytest
 from fastapi.testclient import TestClient
 from main import app
+from utils.db import DataStorage
 
 client = TestClient(app)
+get_room_members_url = (
+    "api/v1/org/3467sd4671a5f5478df56u911/rooms/23dg67l0eba8adb50ca13a24/members"
+)
+
+fake_room_data = {
```

Comment:
well done @pauline-banye 
I think we need to agree on the fake room data structure, since we will be writing our tests in the same file, else our PRs will always be having conflicts.

Reply:
okay @pauline-banye , let's have a meet later and harmonize our thoughts. I will be free from 5pm

Reply:
A meet🤔. Aii. Buzz when you're free.